labgrid/pytestplugin/reporter.py

```diff
@@ -128,29 +129,58 @@ def pytest_runtest_logreport(self, report):
 
 class ColoredStepReporter(StepReporter):
     EVENT_COLORS_DARK = {
+        'expect$': 'blue', # (a lot of output, blend into background)
+        'run': 'magenta',
+        'state_': 'cyan',
+        'transition$': 'yellow',
+        'cycle$|on$|off$': 'white',
+    }
+
+    EVENT_COLORS_LIGHT = {
+        'expect$': 'white', # (a lot of output, blend into background)
+        'run': 'magenta',
+        'state_': 'cyan',
```

Comment:
BTW, I could not find anything that uses `state_`…

Reply:
The GraphStrategy's state methods use that.